lib/src/option.dart

```diff
@@ -94,10 +94,12 @@ class None<A> extends Option<A> {
   @override int get hashCode => 0;
 }
 
-Option<A> none<A>() => new None();
+Option<A> none<A>([dynamic _]) => new None();
```

Comment:
Hehe! Let me digest this for a while. It feels wrong, but I'm not sure why yet :-)